src/EVM/SymExec.hs

```diff
@@ -552,24 +553,23 @@ verify
   -> Maybe (Postcondition RealWorld)
   -> m (Expr End, [VerifyResult])
 verify solvers opts preState maybepost = do
-  liftIO $ putStrLn "Exploring contract"
+  conf <- readConfig
+  when conf.debug $ liftIO $ putStrLn "Exploring"
 
   exprInter <- interpret (Fetch.oracle solvers opts.rpcInfo) opts.maxIter opts.askSmtIters opts.loopHeuristic preState runExpr
-  conf <- readConfig
   when conf.dumpExprs $ liftIO $ T.writeFile "unsimplified.expr" (formatExpr exprInter)
   liftIO $ do
-    putStrLn "Simplifying expression"
+    when conf.debug $ putStrLn "Simplifying expression"
     let expr = if opts.simp then (Expr.simplify exprInter) else exprInter
     when conf.dumpExprs $ T.writeFile "simplified.expr" (formatExpr expr)
 
-    putStrLn $ "Explored contract (" <> show (Expr.numBranches expr) <> " branches)"
+    putStrLn $ "Exploration finished, " <> show (Expr.numBranches expr) <> " branches to check"
```

Comment:
```suggestion
    putStrLn $ "Exploration finished, " <> show (Expr.numBranches expr) <> " branche(s) to check"
```

Super minor, just to be consistent with the rest of the text.

Reply:
Ah, good catch, fixed!